Sources/Xcodeproj/pbxproj().swift

```diff
@@ -176,6 +176,18 @@ func xcodeProject(
     // of its manifest file.
     let manifestFileRef = project.mainGroup.addFileReference(path: "Package.swift", fileType: "sourcecode.swift")
     createPackageDescriptionTarget(for: graph.rootPackages[0], manifestFileRef: manifestFileRef)
+    
+    // Add root level files to the project.
+    func shouldIgnore(_ path: AbsolutePath) -> Bool {
+        return path.basename.hasPrefix(".")
```

Comment:
I think it is useful to ignore dotfiles. (like `.gitignore` or `.DS_Store`).

How do you think?

Reply:
I would like some files to be included like `.swiftlint.yml` but that might be possible with a different flag such as `--include .swiftlint.yml`